client/components/sites-dropdown/style.scss

```diff
@@ -63,6 +63,15 @@
 			color: var(--color-neutral-50);
 		}
 	}
+
+	svg {
+		box-sizing: border-box;
+	}
+
+	&:focus svg {
```

Comment:
We should probably limit this to a11y-focus class (I think its something like `.accessible-focus &:focus svg`, so that this focus style isn't appearing after clicks like it currently is. That class is added higher on the dom when using keyboard nav, and is removed once the user starts clicking.

Also here, we should specify the svg we are targeting for this style, as it is bleeding to others in this area (see the blue box around the globe in my site icon).

![sites-dropdown-styles](https://github.com/user-attachments/assets/fe4cddf6-e043-49e6-9eef-e65ff03d3b19)

Reply:
here is an example of applying a11y focus to something similar: https://github.com/Automattic/wp-calypso/pull/101240

Reply:
Thanks for the catch! This should be fixed in https://github.com/Automattic/wp-calypso/pull/101167/commits/67486239d9460cb2e69c75e126e7899b83a7dbac. How does it look?

Reply:
That looks much better, thanks!